dev/src/main/java/org/eclipse/codewind/intellij/ui/tree/CodewindTreeNodeCellRenderer.java

```diff
@@ -166,8 +166,8 @@ private String getText(CodewindConnection connection) {
     @NotNull
     private String getText(LocalConnection connection) {
         String text = connection.getName();
-        if (connection.getInstallerStatus() != null) {
-            switch (connection.getInstallerStatus()) {
+        if (CodewindManager.getManager().getInstallerStatus() != null) {
```

Comment:
For performance reasons, we should keep the return value of CodewindManager.getManager(), and use it in the several instances of its use in this method.  It would also be good to then keep the value of manager.getInstallerStatus for these two checks.

Reply:
Changes made